src/plugin.js

```diff
@@ -311,6 +311,7 @@ const onPlayerReady = (player, emeError) => {
     // TODO: refactor this plugin so it can use a plugin dispose
     player.on('dispose', () => {
       player.tech_.off('mskeyerror', emeError);
+      keySession.close();
```

Comment:
I don't think `keySession` is defined anywhere in the `plugin.js` file. I think it'll have to be within `eme.js`, or have a dispose callback returned by the handler.